backend/src/main/java/com/ody/notification/domain/FcmTopic.java

```diff
@@ -0,0 +1,25 @@
+package com.ody.notification.domain;
+
+import com.ody.meeting.domain.Meeting;
+import lombok.Getter;
+
+@Getter
+public class FcmTopic {
+
+    private String value;
+
+    public FcmTopic(String rawValue) {
+        this.value = rawValue.replace(":", "-");
+    }
+
+    public FcmTopic(Meeting meeting) {
+        this(build(meeting));
+    }
+
+    private static String build(Meeting meeting) {
+        return new StringBuilder(meeting.getId().toString())
+                .append("_")
+                .append(meeting.getCreatedAt())
+                .toString();
```

Comment:
[제안]

지금도 충분히 좋지만, 가독성을 위해 `+`를 사용하는건 어떤가요?
> JDK1.5 출시 이후부터
자바 컴파일러는 간단한 형식의 '+' 를 이용한 문자열 합치기를 자동적으로 StringBuilder 를 활용한 문자열 붙이기로 변환하기 때문에 '+' 를 사용해도 성능 상의 이슈가 발생하지 않는다고 하네요

[We Don't Need StringBuilder for Simple Concatenation](https://dzone.com/articles/string-concatenation-performacne-improvement-in-ja)
[해석 글](https://choichumji.tistory.com/135)
```suggestion
        return meeting.getId().toString()
                + "_"
                + meeting.getCreatedAt();
```

Reply:
[반영] [4d5f24f](https://github.com/woowacourse-teams/2024-ody/pull/367/commits/4d5f24f79a2ab7e63d332c3c29519e507fda884d)